src/bci_build/package.py

```diff
@@ -2234,6 +2234,9 @@ def _create_php_bci(
         package_list=[
             Package(name, pkg_type=PackageType.BOOTSTRAP)
             for name in (
+                "openSUSE-release-appliance-docker"
+                if os_version == OsVersion.TUMBLEWEED
+                else "sles-release",
```

Comment:
I would suggest to split this out into a separate helper function, as this is otherwise exactly duplicated in 3 places.

Reply:
good idea. actually we can generalize that outside this code path